website/docs/intro.md

```diff
@@ -3,30 +3,34 @@ sidebar_position: 1
 title: "Contributor Guide"
 ---
 
-
 ## We ♥️ Contributions!
-
 `awesome-azd` is a community-friendly resource to help developer discover, create, and share, _azd-template_ resources for streamlining your application development and CI/CD pipelines. It is an _open-source_ effort open for contributions in many ways. 
 
-
 ## Our Custom Issues
-
 The [custom issues](https://github.com/Azure/awesome-azd/issues/new/choose) page shows the different ways in which _anyone_ can contribute to make this project successful for _everyone_. 
-Here are the 4 main ones:
 
- - submit a **resource** for the `awesome-azd/README.md` list
- - submit an **idea** for a template
- - contribute a **requested template** matching an idea
- - contribute a **new template** from your own project or idea
+### Submit a Template Idea
+There are ample ways to combine languages with Azure services and tools to create applications, so if you cannot find a template with architecture that works for you-- you can submit a request for that template. 
```

Comment:
Nit: I'd prefer that we use a casual tone on this site

Maybe just "There are many ways"?